pkg/contracts/constants.go

```diff
@@ -60,10 +60,11 @@ func (t StreamType) Validate() bool {
 type AnnotationType string
 
 const (
-	AnnotationPKI    AnnotationType = "pki"
-	AnnotationSource AnnotationType = "src"
-	AnnotationTLS    AnnotationType = "tls"
-	AnnotationTPM    AnnotationType = "tpm"
+	AnnotationPKI     AnnotationType = "pki"
+	AnnotationHTTPPKI AnnotationType = "httppki"
```

Comment:
Suggestion for the above
```
AnnotationPKIHttp AnnotationType = "pki-http"
```

Reply:
Fixed in ef52a4d46b344e24703744e982ac2fc17aed75c4